class NotFoundObjectError(Exception):
    pass


class ObjectAlreadyCreatedError(Exception):
    pass
